content/root/best-practice-guide.textile

```diff
@@ -19,28 +19,57 @@ h2(#ably-101). Ably 101
 
 Ably Realtime is a cloud-based platform that allows you to easily integrate realtime functionality into your applications. Just plug and play!
 
-h3(#pubsub). Publish and Subscribe (Pub/Sub)
+h3(#auth). Authentication
 
-You can publish data to named "channels":/realtime/channels within Ably's platform and the platform will make sure that all clients subscribed to these channels will receive the data in realtime.
+Ably supports two main methods of authentication: 
 
-h3(#notifications). Notifications
+1. "Basic authentication":/core-features/authentication#basic-authentication 
+2. "Token authentication":/core-features/authentication#token-authentication
 
-Ably offers the "Push Notifications":/general/push feature which allows you to send notifications to your clients even when they are not using your application at the time, or in other words, not connected to Ably. This can be implemented on both mobile devices and browsers. Ably also offers a smart variation known as "Smart Notifications":/general/smart-notifications using which you can make the notifications most effective based on your user's connection state, their active device or even their current context within your application.
+For basic authentication you provide the Ably API key directly when authenticating with Ably. However, basic authentication should not be used on clients (for example a browser client) where the API key would be exposed. In production systems you should always avoid using basic authentication on the client-side - instead use "token authentication":/core-features/authentication#token-authentication using an @authUrl@ or @authCallback@.
```

Comment:
"using either `authCallback` or `authUrl` (the recommended approach)"

If we say this, then it justifies us only talking about token requests later on.

Reply:
Actually, both of those options deal with token requests. We can clarify on a call.